array_find.json

```diff
@@ -26,7 +26,7 @@
         }
     ],
     "returns": {
-        "description": "Returns the index of the first element with the specified value. If no element was found, `null` is returned.",
+        "description": "The index of the first element with the specified value. If no element was found, `null` is a.",
```

Comment:
> is a.

typo?

Reply:
How on earth did that get in here? it should still be `returned`. Thanks for spotting this.